### PR TITLE
add jhbuild

### PR DESCRIPTION
**Pull request by holymonson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/jhbuild/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://gitlab.gnome.org/GNOME/jhbuild/-/archive/{{ version }}/jhbuild-{{ version }}.tar.gz
+  sha256: 5756f50154435c533419acd84671226d1d3cf9f1f6add84e3684340fa7a9e9f1
+
+build:
+  noarch: python
+  number: 0
+  script: |
```
Comment (xhochy) [inline recipes/jhbuild/meta.yaml:15]: Can you put that in a `build.sh`?

File: recipes/jhbuild/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.gnome.org/GNOME/jhbuild/-/archive/{{ version }}/jhbuild-{{ version }}.tar.gz
+  sha256: 5756f50154435c533419acd84671226d1d3cf9f1f6add84e3684340fa7a9e9f1
+
+build:
+  noarch: python
```
Comment (xhochy) [inline recipes/jhbuild/meta.yaml:13]: Compiled recipes cannot be noarch
```suggestion
```

File: recipes/jhbuild/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    ./configure --prefix=$PREFIX
+    make
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - gettext
+    - pkgconfig
+    - make
```
Comment (xhochy) [inline recipes/jhbuild/meta.yaml:27]: Seems to require a C compiler
```suggestion
    - make
    - {{ compiler('c') }}
```

**Comment by holymonson** [comment]
> Non noarch packages should have python requirement without any version constraints.

@xhochy it does require `python >=3.5`, could we ignore this lint?

File: recipes/jhbuild/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - {{ compiler('c') }}
+  host:
+    - python >=3.5
```
Comment (xhochy) [inline recipes/jhbuild/meta.yaml:25]:
```suggestion
    - python
```

File: recipes/jhbuild/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python >=3.5
```
Comment (xhochy) [inline recipes/jhbuild/meta.yaml:27]:
```suggestion
    - python
```

**Review by xhochy — COMMENTED** [review]
As it is compiled, this will generate a package each for each Python version between 3.6 and 3.9, so we don't need a constraint here.

**Review by xhochy — APPROVED** [review]

**Pull request merged**